### PR TITLE
Add preview form example

### PR DESCRIPTION
**Pull request by TheCadien**
#### What's in this PR?

Add missing example for preview Form in Admin Class

### CODE REVIEW

File: bundles/preview/preview-form.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Preview form
```
Comment (alexander-schranz) [inline bundles/preview/preview-form.rst:1]: Not sure about this maybe this should be `preview-form-builder.rst` /cc @danrot what do you think it should be placed.

File: bundles/preview/preview-form.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+============
+
+
+PreviewFormViewBuilder
+**********************
```
Comment (alexander-schranz) [inline bundles/preview/preview-form.rst:6]: We use `--------------` as second headline in our files.

File: bundles/preview/preview-form.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
```
Comment (alexander-schranz) [inline bundles/preview/preview-form.rst:7]: We should add some description text here that the Form with an preview is an own view so an own view builder need to be used and maybe link to https://github.com/sulu/sulu/tree/master/src/Sulu/Bundle/AdminBundle/Resources/js/views/PreviewForm.

File: bundles/preview/preview-form.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        private $viewBuilderFactory;
+
+        public function __construct(ViewBuilderFactoryInterface $viewBuilderFactory)
+        {
+            $this->viewBuilderFactory = $viewBuilderFactory;
+        }
+
+        public function configureViews(ViewCollection $viewCollection): void
+        {
+
```
Comment (alexander-schranz) [inline bundles/preview/preview-form.rst:37]: here should be no empty line.

File: bundles/preview/preview-form.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            $editFormView = $this->viewBuilderFactory->createResourceTabViewBuilder(static::EVENT_EDIT_FORM_VIEW, '/events/:id')
+                ->setResourceKey(Event::RESOURCE_KEY)
+                ->setBackView(static::EVENT_LIST_VIEW);
+
+            $viewCollection->add($editFormView);
+
+            $editDetailsFormView = $this->viewBuilderFactory->createPreviewFormViewBuilder(static::EVENT_EDIT_FORM_VIEW . '.details', '/details')
```
Comment (alexander-schranz) [inline bundles/preview/preview-form.rst:44]: The optional `setPreviewCondition` should also be here:

```suggestion
            $editDetailsFormView = $this->viewBuilderFactory->createPreviewFormViewBuilder(static::EVENT_EDIT_FORM_VIEW . '.details', '/details')
                ->setPreviewCondition('id != null') // this is an optional condition when the preview should be shown
```

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
@TheCadien Thank you for adding this to the docs:

Some small change request. /cc @danrot do you have any additional change requests?

**Comment by TheCadien** [comment]
The https://github.com/sulu/sulu-docs/blob/master/bundles/preview/index.rst file isnt existing in 2.0 
Should i change the PR to 2.1 ?

**Comment by alexander-schranz** [comment]
@TheCadien just add:

```rst

.. toctree::
    :maxdepth: 2
    preview-form
```

at the end of the exist [2.0 index.rst](https://github.com/sulu/sulu-docs/blob/2.0/bundles/preview/index.rst) file.

File: bundles/preview/preview-form-builder.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
+Preview form
+============
+
+
+PreviewFormViewBuilder
+----------------------
```
Comment (alexander-schranz) [inline bundles/preview/preview-form-builder.rst:6]: Would this change to:

```suggestion
PreviewFormViewBuilder
======================

Add some description text here as commented before.
```

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@TheCadien Thank you!

**Pull request merged**